x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/operator/OperatorOnlyRegistry.java

```diff
@@ -23,21 +23,18 @@ public interface OperatorOnlyRegistry {
     OperatorPrivilegesViolation check(String action, TransportRequest request);
 
     /**
-     * Checks to see if a given {@link RestHandler} is subject to operator-only restrictions for the REST API.
+     * Checks to see if a given {@link RestHandler} is subject to full restrictions for the REST API.
```

Comment:
I would suggest to nuke most of this doc here, since this is for the registry which is only called when the user is not an operator. Perhaps replace it with something like :  

```
    /**
     * This method is only called if the user is not an operator. Implementations should fail the request if the RestRequest is not allowed to proceed by throwing an {@link org.elasticsearch.ElasticsearchException}. If the request should be handled by the associated RestHandler, then this implementations should do nothing. 
     */
```

Reply:
👍 less is more